airbyte-webapp-e2e-tests/cypress/commands/common.ts

```diff
@@ -16,48 +16,48 @@ Cypress.Commands.add("fillTestLocalJsonForm", (name) => {
   
   cy.get("input[name=name]").type(name);
   cy.get("input[name='connectionConfiguration.destination_path']").type("/local");
-})
+}
 
-Cypress.Commands.add("openSourcePage", () => {
+export const openSourcePage = () => {
   cy.intercept("/api/v1/sources/list").as("getSourcesList");
   cy.visit("/source");
   cy.wait("@getSourcesList");
-})
+}
 
-Cypress.Commands.add("openDestinationPage", () => {
+export const openDestinationPage = () => {
```

Comment:
Curious about the choice to not add these as Cypress Commands.  Is it just for clarity or does it improve the TypeScript support?

Reply:
If you look at [the cypress documentation](https://docs.cypress.io/guides/tooling/typescript-support#Types-for-custom-commands), the process just adds extra overhead in that for every command that you add you also need to add a type definition of the command, so you then need to type in two places but also know how to properly type it so that cypress is happy. I don't see a big advantage in doing this.

Reply:
I also haven't had the best experience how the typing actually works (if different commands expect different parameter amounts, cypress has some bad typing inside causig issues). The only benefit I'd see having those as commands is that the cypress runner UI would clearly show it as "one command" in the list and thus it's clearer that this was a "command step" instead of unfolding it into it's individual steps of the command (as if they'd be part of the actual test suit), potentially making it a tiny bit harder seeing exactly where in the test method the runner currently is.

That said I don't think that's a strong enough argument to not go the path of having this methods imported as TS methods as this PR changes, thus no strong feeling from me to either direction and happy to have it changed as in this PR.